app/src/main/res/navigation/nav.xml

```diff
@@ -57,8 +57,8 @@
         android:label="fragment_get_started"
         tools:layout="@layout/fragment_get_started" >
         <action
-            android:id="@+id/action_getStartedFragment_to_dashboard"
-            app:destination="@id/dashboard" />
+            android:id="@+id/action_getStartedFragment_to_addTask"
+            app:destination="@id/addTask" />
```

Comment:
Why is the user going from Get Started Fragment to add task. As this is an internal application the task should already be there even if he/she is a new user.

Reply:
We discussed this previously, the user should be directed to the add task page so they can create their first task easily